### DISCUSSION TITLE
developing-rmmv-plugins/index/

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#2] [Announcements]
# developing-rmmv-plugins/index/

如果你已经熟练地掌握了RPGMV这款软件的运用，你多少会感觉到：这款软件的局限性太大了，很多东西实现不了。为了突破这些局限性，不少游戏开发者深入项目源码，用编写插件的方式来实现更多样化的功能。 在这里，将会涉及到很多复杂的内容，会涉及到 RPGMV 的插件开发和PixiJS的源码分析。建议读者具有以下的编程基础： 懂得基本的C语言编程技术。; 懂得函数...

https://www.ruan-cat.com/developing-rmmv-plugins/index/

**Comment by ruan-cat** [comment]
很高兴可以看到本内容，这是一条测试性质的文本。
